### DISCUSSION TITLE
Eagler Reborn Status Update 2: Electric Boogaloo

### DISCUSSION THREAD

**Discussion by leahanderson1** [EaglerReborn/support#17] [Announcements]
I'm 90% no one will see this, but I just wanna say that Eagler Reborn 1.0 has been delayed due to the recent release of Eaglercraft u21, as I will (finally!) do a rewrite for Eagler Reborn

## stuff i had done before the rewrite

- fixed the fullscreen bug
- ported those old API classes to the Plugin API

Anyway, expect Eagler Reborn 1.0 to be coming out Q2 of 2024.

**Comment by wxnnvs** [comment]
What does Q2 mean ?
Besides that, nice!

**Reply by leahanderson1** [reply]
quarter 2.

**Reply by wxnnvs** [reply]
Alr thx
